src/componets/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import { Post } from '../../types/Post';
+
+import './PostInfo.scss';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = (props) => {
+  const { post } = props;
+  const {
+    title, user, body, comments,
+  } = post;
+
+  return (
+    <li className="postInfo">
+      <h2>
+        {title}
+      </h2>
+      {
+        user && (
+          <h3>
+            <UserInfo userInfo={user} />
+          </h3>
```

Comment:
move that headings to the UserInfo component

Reply:
I do not understand why do this?